resource/plugins/unreal/python/publisher/context/widget/unreal_project_publisher_context_options.py

```diff
@@ -0,0 +1,395 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+from Qt import QtWidgets, QtCore, QtGui
+
+import unreal
+
+import ftrack_api
+
+from ftrack_connect_pipeline_qt import plugin
+
+from ftrack_connect_pipeline_qt.plugin.widget import BaseOptionsWidget
+from ftrack_connect_pipeline_qt.plugin.widget.context import StatusSelector
+from ftrack_connect_pipeline_qt.ui.utility.widget.context_selector import (
+    ContextSelector,
+)
+from ftrack_connect_pipeline_qt.ui.utility.widget import line
+from ftrack_connect_pipeline_qt.ui.utility.widget.asset_selector import (
+    AssetSelector,
+)
+from ftrack_connect_pipeline_qt.utils import BaseThread
+from ftrack_connect_pipeline_qt.ui.utility.widget import dialog
+from ftrack_connect_pipeline_qt.ui.utility.widget.entity_info import EntityInfo
+
+from ftrack_connect_pipeline_unreal.utils import (
+    custom_commands as unreal_utils,
+)
+
+
+class UnrealProjectPublisherContextOptionsWidget(BaseOptionsWidget):
```

Comment:
UnrealRootPublisherContextOptionsWidget

Reply:
Then also change it for the plugin as well